src/OrbitGl/TimerTrack.h

```diff
@@ -29,6 +29,25 @@
 class OrbitApp;
 class TextRenderer;
 
+namespace internal {
+struct DrawData {
```

Comment:
Would it make sense to group data that is a bit related here? The order here seems random. Also, if you order by size, the size of the struct may be smaller, as you can avoid padding (likely with the single bool in the middle).

Reply:
Done.